app/models/manageiq/providers/autosde/storage_manager/physical_storage.rb

```diff
@@ -53,11 +53,12 @@ def raw_update_physical_storage(options = {})
 
   def self.raw_create_physical_storage(ext_management_system, options = {})
     sys_to_create = ext_management_system.autosde_client.StorageSystemCreate(
-      :password       => options['password'],
-      :user           => options['user'],
-      :system_type    => PhysicalStorageFamily.find(options['physical_storage_family_id']).name,
-      :management_ip  => options['management_ip'],
-      :storage_family => "ontap_7mode"
+      :password                       => options['password'],
```

Comment:
If we aren't `ManageIQ::Password.encrypt`'ing the password here we need to since the `options` will end up in the queue and can be seen in the logs.  Then you will need to `ManageIQ::Password.try_decrypt(options['password'])` here

Reply:
I will open a ticket for the next sprint to fix this and encrypt the password in all the relevant places